styles/content/typography/body/styles.less

```diff
@@ -2,45 +2,40 @@
   /* body */
 
   html,
-  body,
-  p {
+  body {
     .element-text-size(body, null);
     .element-text-style(body);
     display: block;
 
     & when (@body-emphasis-enabled) {
 
-      &.emphasis,
-      .emphasis {
+      span.emphasis {
```

Comment:
Why add the `span` selector here? I suggest we keep this class agnostic and leave it as a class. In DC/OS UI, I know for sure we use `emphasis` on `a` elements as well.

edit: I can see that you've moved the element-agnostic declaration to [line 66](https://github.com/mesosphere/cnvs/pull/97/files#diff-95cf32a51dd729266ef9f0012d6992e4R66), so I'm confused about the `span.emphasis` selector here.